app/views/media_objects/_thumbnail.html.erb

```diff
@@ -14,7 +14,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 ---  END LICENSE_HEADER BLOCK  ---
 %>
 
-<div id="thumbnail-button" style="display: inline-block;">
+<div id="thumbnail-button" style="display: inline-block">
```

Comment:
And similarly here we could use `d-inline-block` Bootstrap class instead using inline CSS? I haven't really used this before, so not sure it gives the same appearance. Docs: https://getbootstrap.com/docs/4.0/utilities/display/